cloud/types.go

```diff
@@ -202,6 +202,8 @@ type (
 		GithubActionsDeploymentTriggeredBy string `json:"github_actions_triggered_by,omitempty"`
 		GithubActionsRunID                 string `json:"github_actions_run_id,omitempty"`
 		GithubActionsRunAttempt            string `json:"github_actions_run_attempt,omitempty"`
+		GithubActionsWorkflowName          string `json:"github_actions_workflow_name,omitempty"`
+		GithubActionsWorkflowPath          string `json:"github_actions_workflow_path,omitempty"`
```

Comment:
This should be called ref as it is not a path but a complete reference.

Reply:
Thanks!